rc/extra/gpg.kak

```diff
@@ -0,0 +1,57 @@
+# Enable kakoune to open, edit, write files encrypted
+# using [GNU privacy guard](https://www.gnupg.org/)
+
+# options {
+declare-option -docstring 'Set this to true to add a signature when saving the file.' bool gpg_sign false
+declare-option -docstring 'Encrypt for this user id. If not set, it is a assumed that a default was configured in ~/.gnupg/gpg.conf.' str gpg_recipient 
+declare-option -docstring 'Set this to a secure deletion command e.g. shred(1).' str gpg_rmcmd 'rm -f'
+# }
+
+hook -group gpg global BufOpenFile .*\.gpg %{
+    set-option buffer autoreload no
+    %sh{
+        f="${kak_buffile%.gpg}"
+        umask 066
+        gpg --output "$f" --yes --decrypt "${kak_buffile}" 2>/dev/null
```

Comment:
The return code should be checked here, and an error printed if the decryption couldn't happen. Also let `gpg` print errors to the debug buffer.

Reply:
However, I can't seem to get messages to display in the mode line. :disappointed: